flutter/android/app/src/main/kotlin/com/carriez/flutter_hbb/InputService.kt

```diff
@@ -252,9 +261,27 @@ class InputService : AccessibilityService() {
         }
     }
 
+    @RequiresApi(Build.VERSION_CODES.N)
+    fun onKeyEvent(data: ByteArray) {
+        val keyEvent = KeyEvent.parseFrom(data);
+        getInputMethod()?.let { inputMethod ->
```

Comment:
getInputMethod added in [API level 33](https://developer.android.com/guide/topics/manifest/uses-sdk-element#ApiLevels)

Reply:
Good point

Reply:
You may have some modification to make it compatible with lower level target device.

Reply:
I checked the wrong field, I thought we were using API 33. My bad. So I just keep this for higher version and add the old code back for lower API?

Reply:
I am not familiar with kotlin about how to call higher API on high target device but still can run on low target device.

Reply:
My android device may be the lower version. The input does not work now, and it reports:
```
D/ffi     ( 3011): librustdesk::server::connection: call_main_service_key_event fail:Java exception was thrown
```

Reply:
Could you try it again? I added old code back with a version check.

Reply:
The keyboard input works now.